webapp/src/components/dataset_input/TextDatasetInput.vue

```diff
@@ -10,20 +10,27 @@
 import { Set } from "immutable";
 import { ref, watch } from "vue";
 
-import type { Text } from "@epfml/discojs";
-import { Dataset } from "@epfml/discojs";
+import type { Text, Task } from "@epfml/discojs";
+import { Dataset, models } from "@epfml/discojs";
 import { loadText } from "@epfml/discojs-web";
 
 import DatasetInput from "./DatasetInput.vue";
 import FileSelection from "./FileSelection.vue";
 
+const { task } = defineProps<{ task: Task }>()
```

Comment:
[destructuring props only works in vue >= 3.5](https://vuejs.org/guide/components/props.html#reactive-props-destructure) but we are using vue 3.4 here